src/System.Memory/pkg/System.Memory.pkgproj

```diff
@@ -5,8 +5,8 @@
     <ProjectReference Include="..\ref\System.Memory.csproj">
       <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.Memory.csproj">
-      <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+    <ProjectReference Include="..\src\System.Memory.builds">
+      <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Go ahead and try to remove it. If it's actually needed then you will get a build error and message about it. But judging by that other project, you shouldn't need it.